src/types/types.tsx

```diff
@@ -0,0 +1,105 @@
+type filling = {
+  name: string;
+  tastes: {
+    flavor: string;
+    amount: number;
+  }[];
+  powers: {
+    type: string;
+    amount: number;
+  }[];
+  types: {
+    type: string;
+    amount: number;
+  }[];
+  imageUrl: string;
+  pieces: number;
+};
+
+type condiment = {
+  name: string;
+  tastes: {
+    flavor: string;
+    amount: number;
+  }[];
+  powers: {
+    type: string;
+    amount: number;
+  }[];
+  types: {
+    type: string;
+    amount: number;
+  }[];
+  imageUrl: string;
+};
+
+type power = {
+  type: string;
+  amount: number;
+  modded?: boolean;
+  boosted?: boolean;
+};
+
+type taste = {
+  flavor: string;
+  amount: number;
+};
+
+type type = {
+  type: string;
+  amount: number;
+};
+
+type summation = {
```

Comment:
I just matched best I could. Not too sure what dropped / overflow is though. Seems new compared to yesterday.